interop/quic-openssl-docker/run_endpoint.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Just curious, what are thoughts on setting options like `set -euxo pipefail`?

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
Yeah, I can set those, though generally I'm a bit hesitant to set -x on an in-production system, as it clutters the logs quite a bit

Reply:
added